### ISSUE TITLE
AAPXS for host extension

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/aap-core#181]
When we have introduced AAPXS, there was no host extension (such as `plugin-info`). Currently host extensions are implemented by host directly, which means, every host needs to implement the extensions.

There are other extensions like `aap_host_parameters_extension_t` that needs to send the host the notifications. In realtime mode, they also need to be sent via AAPXS SysEx8 and that should be implemented by AAPXS developers.

Currently `AAPXSFeature` has only `as_proxy()` which is only for plugin API. That needs o have another proxy for host extensions, and all relevant implementation classes need to implement the additional features.

**Comment by atsushieno** [comment]
It is being implemented in `aapxs-v080` branch.

**Comment by atsushieno** [comment]
09dcd54c implements host AAPXS. We can close it once `aapxs-v080` is merged into main.

**Comment by atsushieno** [comment]
`aapxs-v080` is now merged into main.
